internal/backend/gs/gs.go

```diff
@@ -205,7 +210,7 @@ func (be *Backend) Save(ctx context.Context, h restic.Handle, rd restic.RewindRe
 	if err := h.Valid(); err != nil {
 		return err
 	}
-
+	var w *storage.Writer
```

Comment:
Why is the variable defined here and not around line 247 (before / after the base64 decoding?)

Reply:
It's placement is just from my familiarity with C over Go. I'll move the declaration.